src/main/java/com/slimebot/games/games/Wordchain.java

```diff
@@ -0,0 +1,210 @@
+
+        scheduledFuture = Main.executor.schedule(() -> {
+            kickPlayer(this.player.get(playerTurn), getChannel().sendMessage(":x: <@" + player +"> ist Ausgeschieden weil die Zeit abgelaufen ist!"));
+            nextTurn();
+        }, seconds, TimeUnit.SECONDS);
+    }
+
+    private void damage(long player) {
+        short lives = playerLives.get(player);
+        if(lives <= 1) {
+            kickPlayer(player, getChannel().sendMessage(":x: <@" + player +"> ist Ausgeschieden!"));
+            playerLives.remove(player);
+            return;
+        }
+        lives--;
+        playerLives.put(player, lives);
+
+        getChannel().sendMessage(":x: <@" + player + "> hat ein Fehler gemacht und hat nur noch **" + lives + "** Versuche!").queue();
+    }
+
+    private void kickPlayer(long player, RestAction<?> restAction) {
+        restAction.queue();
+
+        leave(player);
+    }
+
+    public MessageEmbed updateJoinEmbed(MessageEmbed embed) {
+        return new EmbedBuilder()
+                .setColor(Main.embedColor(String.valueOf(guildId)))
+                .setTitle(embed.getTitle())
+                .setDescription("Um zu erfahren wie \"Wortkette\" funktioniert nutze ```/wordchain explanation```") // TODO
+                .addField("Spielleiter:in:", "<@" + gameMaster + ">", true)
+                .addField(":timer: Timeout:", seconds + "s", true)
+                .addField(":x: Max Fehler:", lives + " Fehler", true)
+                .addField("Spieler:", player.stream().map(p -> ("<@" + p + ">")).collect(Collectors.joining("\r\n")), true)
+                .setFooter("GameID: " + uuid)
+                .setTimestamp(Instant.now())
+                .build();
+    }
+
+    @Override
+    public void onMessageReceived(MessageReceivedEvent event) {
+        if(status != GameStatus.PLAYING) return;
+        if (event.getChannel() != getChannel()) return;
+        long memberID = event.getMember().getIdLong();
+        if (memberID != player.get(playerTurn)) return;
+
+        String content = event.getMessage().getContentRaw();
+
+        if(lastWord != null) {
+            if (content.split(" ").length > 1 ||
+                    Character.toLowerCase(content.toCharArray()[0]) != Character.toLowerCase(lastWord.charAt(lastWord.length()-1)) ||
+                    words.contains(content.split(" ")[0].toLowerCase())) {
+                damage(memberID);
+            } else {
+                lastWord = content.split(" ")[0].toLowerCase();
+                words.add(lastWord);
+            }
+        } else {
+            lastWord = content.split(" ")[0].toLowerCase();
+            words.add(lastWord);
+        }
+        nextTurn();
+    }
+
+    @Override
+    public void onButtonInteraction(ButtonInteractionEvent event) {
+        String id = event.getButton().getId();
```

Comment:
`String[] id = event.getComponentId().split(":")`
Return `if(!id[0].equals(uuid.toString()))`
put `id[1]` into the switch